docs/component-guides/kserve.md

```diff
@@ -0,0 +1,70 @@
+# Use S3 as storage Uri for KFServing/KServe
+  
+## Motivation  
+When deploying a model server, you can specify in the `storageUri` the S3 path to your model.  
+If the S3 bucket is private, KFServing/KServe will need to be authorized to download your model.  
+To do so, we can leverage the fact that KFServing/KServe supports Kubernetes [Service Accounts](https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/#:~:text=A%20service%20account%20provides%20an,recommended%20by%20the%20Kubernetes%20project.).  
+
+## Overview  
+The goal is to create a Service Account that has access to the bucket and then specify this Service Account in the inference service definition to allow KFServing/KServe to download the model.    
+The Service Account will reference the secret with AWS credentials created during the installation of Kubeflow, more precisely, it will reference the secret `mlpipeline-minio-artifact` created by the Secrets Manager.  
+  
+## 1.0 Prerequisites  
+  
+1. The secrets were setup for the namespace you want to deploy your model. This can be done by following [this guide](./notebooks.md#setup-the-secrets-access)
+  
+## 2.0 Setup  
```

Comment:
everything is very well described in kserve docs here: https://kserve.github.io/website/modelserving/storage/s3/s3/. What value are we adding by having these instructions here? Trying to avoid duplication because there is a high chance one of them can eventually get outdated.

I think we should give a brief summary and redirect to Kserve documentation

Reply:
@surajkota I was not sure if we should put this in AWS repo or in KServe doc.  
It seems like dsun suggested we update KServe doc (for context see https://kubeflow.slack.com/archives/CH6E58LNP/p1647567321171729?thread_ts=1647566004.793539&cid=CH6E58LNP)  
  
So I think you're right, we should keep it simple here and redirect to KServe doc and update KServe doc with whatever approach is taken.